docs/services/thorlabs_mff101.rst

```diff
@@ -29,4 +30,6 @@ Commands
 
 Datastreams
 -----------
-``position``: the current (commanded) position of the flip mount. This can be set to either 1 or 2, indication position 1 or 2. Other values will be silently ignored.
+``current_position``: the current position of the flip mount. This can be set to either 1 or 2, indicating position 1 or 2. Other values will be silently ignored.
```

Comment:
```suggestion
``current_position``: The current position of the flip mount. This can be set to either 1 or 2, indicating position 1 or 2. Other values will be silently ignored.
```